_includes/pbc/all/install-features/202212.0/install-the-product-feature.md

```diff
@@ -0,0 +1,353 @@
+
+```php
+<?php
+
+namespace Pyz\Zed\Event;
+
+use Spryker\Zed\Event\EventDependencyProvider as SprykerEventDependencyProvider;
+use Spryker\Zed\ProductPageSearch\Communication\Plugin\Event\Subscriber\ProductConcretePageSearchProductImageEventSubscriber;
+
+class EventDependencyProvider extends SprykerEventDependencyProvider
+{
+    public function getEventSubscriberCollection()
+    {
+        $eventSubscriberCollection = parent::getEventSubscriberCollection();
+        $eventSubscriberCollection->add(new ProductConcretePageSearchProductImageEventSubscriber());
+
+        return $eventSubscriberCollection;
+    }
+}
+```
+
+**src/Pyz/Zed/ProductPageSearch/ProductPageSearchDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductPageSearch;
+
+use Spryker\Zed\ProductPageSearch\Communication\Plugin\PageDataExpander\ProductImageProductConcretePageDataExpanderPlugin;
+use Spryker\Zed\ProductPageSearch\Communication\Plugin\PageMapExpander\ProductImageProductConcretePageMapExpanderPlugin;
+use Spryker\Zed\ProductPageSearch\ProductPageSearchDependencyProvider as SprykerProductPageSearchDependencyProvider;
+
+class ProductPageSearchDependencyProvider extends SprykerProductPageSearchDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductPageSearchExtension\Dependency\Plugin\ProductConcretePageMapExpanderPluginInterface[]
+     */
+    protected function getConcreteProductPageMapExpanderPlugins(): array
+    {
+        return [
+            new ProductImageProductConcretePageMapExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\ProductPageSearchExtension\Dependency\Plugin\ProductConcretePageDataExpanderPluginInterface[]
+     */
+    protected function getProductConcretePageDataExpanderPlugins(): array
+    {
+        return [
+            new ProductImageProductConcretePageDataExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\ProductPageSearch\Communication\Plugin\Publisher\Product\ProductConcretePageSearchWritePublisherPlugin;
+use Spryker\Zed\ProductPageSearch\Communication\Plugin\Publisher\ProductConcretePublisherTriggerPlugin;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface>
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return [
+            new ProductConcretePublisherTriggerPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure the following:
+1. After executing the `console sync:data product_concrete` command, product data including images is synced to Elasticsearch product concrete documents.
+2. When a product or its images updated by Zed UI product data including images is synced in respective Elasticsearch product concrete documents.
```

Comment:
Rewrite the sentence. It's hard to understand.

Reply:
This text wasn't added by me. It comes from the older page version.
As I see it was adjusted by you in [this](https://github.com/spryker/spryker-docs/pull/1214/commits/f07b697b537e798100e530d6c420f7cebb2fa592) commit